app-frontend/src/app/components/projects/annotateSidebarItem/annotateSidebarItem.controller.js

```diff
@@ -71,4 +69,32 @@ export default class AnnotateSidebarItemController {
     onQaCheck(annotation, qa) {
         this.onQaChecked({annotation, qa});
     }
+
+    onLabelNameChange() {
+        if (this.labelNameInput.length >= 3) {
+            this.matchLabelName(this.labelNameInput);
+        }
```

Comment:
I think we need an else here to hide the auto-suggest when we don't have enough characters. The other option is to add a length check to the `ng-if` of the suggestion container.